integration-tests/db/db.js

```diff
@@ -36,6 +36,7 @@ const getPayload = reportId => {
     })
     .then(result => result.rows[0].form)
 }
+const seedReports = async reports => Promise.all(reports.map(report => seedReport(report)))
```

Comment:
as you could probably expect, this massively reduces the speed of creating lots of reports.